website/docs/r/netapp_account.html.markdown

```diff
@@ -0,0 +1,86 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_netapp_account"
+sidebar_current: "docs-azurerm-resource-netapp-account"
+description: |-
+  Manage Azure NetApp Account instance.
+---
+
+# azurerm_netapp_account
+
+Manage Azure NetApp Account instance.
+
+
+## NetApp Account Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "acctestRG"
+  location = "Eastus2"
+}
+
+resource "azurerm_netapp_account" "example" {
+  name                = "acctestnetappaccount"
+  resource_group_name = "${azurerm_resource_group.example.name}"
+  location            = "${azurerm_resource_group.example.location}"
+
+  active_directories {
+    username            = "aduser"
+    password            = "aduserpwd"
+    smb_server_name     = "SMBSERVER"
+    dns                 = "1.2.3.4"
+    domain              = "westcentralus.com"
+    organizational_unit = "OU=FirstLevel"
+  }
+
+  tags = {
+    env = "test"
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `name` - (Required) The name of the NetApp Account. Changing this forces a new resource to be created.
+
+* `resource_group_name` - (Required) The resource group name of the NetApp Account. Changing this forces a new resource to be created.
+
+* `location` - (Required) Resource location. Changing this forces a new resource to be created.
+
+* `active_directories` - (Optional) One or more `active_directories` block defined below.
+
+* `tags` - (Optional) Resource tags. Changing this forces a new resource to be created.
+
+---
+
+The `active_directories` block supports the following:
+
+* `dns` - (Required) Comma separated list of DNS server IP addresses for the Active Directory domain.
+
+* `domain` - (Required) Name of the Active Directory domain.
+
+* `organizational_unit` - (Optional) The Organizational Unit (OU) within the Windows Active Directory.
+
+* `password` - (Required) Plain text password of Active Directory domain administrator.
+
+* `smb_server_name` - (Required) NetBIOS name of the SMB server. This name will be registered as a computer account in the AD and used to mount volumes.
+
+* `username` - (Required) Username of Active Directory domain administrator.
```

Comment:
Is this the ad admin used to add the netapp to the domain? could we clarify the docs here?

Reply:
Fixed